vivisect/symboliks/tests/test_intel.py

```diff
@@ -3,12 +3,24 @@
 import vivisect.symboliks.archs.i386 as i386sym
 import vivisect.symboliks.analysis as v_s_analysis
 
-from vivisect.symboliks.common import Var, Const, cnot
-from vivisect.symboliks.effects import ConstrainPath
+from vivisect.symboliks.common import Arg, Var, Const, cnot
+from vivisect.symboliks.effects import ConstrainPath, WriteMemory
 
 import vivisect.tests.helpers as helpers
 
 
+def getPathFromOpcodes(vw, opcodes):
```

Comment:
this feels like something that might be a part of a utility module.  i can see things i'd want this for outside this context.